source/Engine.cpp

```diff
@@ -2290,24 +2290,24 @@ void Engine::HandleMouseClicks()
 // Determines alternate mouse turning, setting player mouse angle, and right-click firing weapons.
 void Engine::HandleMouseInput(Command &activeCommands)
 {
+	bool rightMouseButtonHeld = false;
+	int mousePosX, mousePosY;
```

Comment:
```suggestion
	int mousePosX
	int mousePosY;
```

Reply:
I'm going to argue on this one, is there a source for this? I think having these particular variables being on the same line helps with readability.

Reply:
We don't have any explicit restriction against it in the code style guide, but it's definitely a rare pattern in our codebase. I can only find two examples of defining multiple variables on the same line.

https://github.com/endless-sky/endless-sky/blob/b5369dc8901832c4685c1e4cf18758d6e636c4b5/source/GameWindow.cpp#L318

https://github.com/endless-sky/endless-sky/blob/b5369dc8901832c4685c1e4cf18758d6e636c4b5/source/MapPanel.cpp#L1217

Reply:
The GameWindow example is also interfacing with SDL, which can't be done in any other way without looking bad :P

Reply:
Seeing as this pattern is used elsewhere, I'm fine with not changing it here and figuring out later if we want to bother dealing with this in the style guide.